.ghal.rules.json

```diff
@@ -53,6 +53,36 @@
           }
         },
         "technology": {
+          "(?i)csharp-advanced-concepts$": {
+            "labels-add": ":card_file_box: Technology - C# Advanced concepts"
+          },
+          "(?i)csharp-async$": {
+            "labels-add": ":card_file_box: Technology - Async Task"
+          },
+          "(?i)csharp-diagnostics$": {
+            "labels-add": ":card_file_box: Technology - C# / VB diagnostics"
+          },
+          "(?i)csharp-fundamentals$": {
+            "labels-add": ":card_file_box: Technology - C# Fundamentals"
+          },
+          "(?i)csharp-get-started$": {
+            "labels-add": ":card_file_box: Technology - C# Get Started"
+          },
+          "(?i)csharp-language-reference$": {
+            "labels-add": ":card_file_box: Technology - C# Language Reference"
+          },
+          "(?i)csharp-linq$": {
+            "labels-add": ":card_file_box: Technology - LINQ"
+          },
+          "(?i)csharp-null-safety$": {
+            "labels-add": ":card_file_box: Technology - C# null safety"
```

Comment:
```suggestion
            "labels-add": ":card_file_box: Technology - C# Null Safety"
```

Reply:
I cross-checked with the label (the keys here must match the label text). The correct capitalization is "Null safety"

We can discuss changing in both places, but that's correct for now.